lamp/courses.php

```diff
@@ -1,20 +1,58 @@
 <?php
-    $page_title = "Courses";
+$page_title = "My Courses";
+$center_page = true;
 ?>
 <?php require_once("lib/page-setup.php") ?>
 <?php require_once("lib/course-handler.php") ?>
 <?php include('common/header.php'); ?>
-<?php 
-function course_list_show_course($course) { ?>
-    <li><a href="channels.php?ch_id=<?php echo(urlencode(htmlspecialchars($course->course_id)));?>"><?php echo(htmlspecialchars($course->course_code)); ?>-<?php echo(htmlspecialchars($course->section_number)); ?></a></li>
-<?php } ?>
 <?php
-    $courses = CourseMembership::get_user_courses($_SESSION['uid']);
-    
+function course_list_show_course($course, $stu_only = null)
+{
+    if ($stu_only === null || ($stu_only === false && $course->is_instructor()) || ($stu_only == true && !$course->is_instructor())) { ?>
+        <a href="channels.php?ch_id=<?php echo (urlencode(htmlspecialchars($course->course_id))); ?>" class="list-group-item list-group-item-action"><?php echo (htmlspecialchars($course->course_code)); ?>-<?php echo (htmlspecialchars($course->section_number)); ?></a>
+<?php
+    }
+}
+?>
+<?php
+$courses = CourseMembership::get_user_courses($_SESSION['uid']);
+$course_types = ['instructor' => 0, 'student' => 0];
+foreach ($courses as $course) {
+    if ($course->is_instructor()) $course_types['instructor']++;
+    else $course_types['student']++;
+}
+// $courses[0]->role = 0;
+
 ?>
 Courses<br />
```

Comment:
Don't think the header should be left as plain text. Even just a larger font, bold, or different color to set it aside from the other text will make it look more professional.